src/java/org/apache/cassandra/utils/JVMStabilityInspector.java

```diff
@@ -58,11 +59,32 @@ private JVMStabilityInspector() {}
      */
     public static void inspectThrowable(Throwable t) throws OutOfMemoryError
     {
-        inspectThrowable(t, true);
+        inspectThrowable(t, null, true, true);
```

Comment:
most examples I see calling this method happen in the current thread, so thinking we should use the current thread as the default rather than null.